### DISCUSSION TITLE
official helm charts?

### DISCUSSION THREAD

**Discussion by sigJoe** [conductor-oss/conductor#106] [Q&A]
Is there a plan to create an official repo for `conductor-oss/conductor-helm-charts`, or are people encouraged to work off of [orkes-io/orkes-helm-charts](https://github.com/orkes-io/orkes-helm-charts)?

**Comment by dradoaica** [comment]
First, thank you for the product and your hard work!

Secondly, I played around with creating a helm chart: https://github.com/dradoaica/helm-charts/tree/main/charts/conductor-oss-conductor.
